Overview.bs

```diff
@@ -691,23 +682,24 @@ when a set (ie. code points, feature tags, design space, child entries) is not s
 
 The inputs to this algorithm are:
 
-*   <var>Patch URI</var>: A [[rfc3986#section-4.1|URI Reference]] identifying the patch file to load. As a URI reference this may be a
-     relative path.
+*   <var>patch URL string</var>: A [[whatwg-url|URL]] string ([[!UTF-8]] encoded) identifying the patch file to load. It  may be a
+     [[whatwg-url#relative-url-string|relative]] URL.
 
-*   <var>Initial Font URI</var>: An [[rfc3986#section-4.3|absolute URI]] which identifies the initial incremental font that the
-     patch URI was derived from.
+*  <var>initial font URL</var>: a [[whatwg-url#concept-url|URL]] which identifies the location of the initial incremental
+    font that <var>font subset</var> was derived from.
 
 The algorithm outputs:
 
-*   <var>patch file</var>: the content (bytes) identified by <var>Patch URI</var>.
+*   <var>patch file</var>: the content (bytes) identified by <var>patch URL string</var>.
 
 The algorithm:
 
-1.  Perform [[rfc3986#section-5|reference resolution]] on <var>Patch URI</var> using <var>Initial Font URI</var> as the base URI to
-     produce the <var>target URI</var>.
+1.  Parse <var>patch URL string</var> into a [[whatwg-url#concept-url|URL Record]] using [[whatwg-url#url-parsing]].
+     <var>initial font URL</var> is the [[whatwg-url#concept-base-url|base URL]] and encoding is UTF-8. Store the result
+     in <var>target URL</var>.
 
-2.  Retrieve the contents of <var>target URI</var> using the fetching capabilities of the implementing user agent. For web browsers,
-     [[fetch]] should be used. When using [[fetch]] a request for patches should use the same CORS settings as the initial request for
+2.  Retrieve the contents of <var>target URL</var> using the fetching capabilities of the implementing user agent. For web browsers,
+     [[FETCH]] should be used. When using [[FETCH]] a request for patches should use the same CORS settings as the initial request for
```

Comment:
why does this PR remove "fetch" from the bibliography? Is it because the specref one is more recent?

Reply:
Yeah, I prefer to use the specref's when they are available since they stay up to date unlike our hardcoded ones.